opcua/server/history.py

```diff
@@ -90,16 +37,17 @@ def save_node_value(self, node, datavalue):
             data = data[-count:]
 
     def read_node_history(self, node, start, end, nb_values):
+        node_id = node.NodeId
```

Comment:
It's because in that part you don't get a UA Node. You get a ua.HistoryReadValueId object. In that object node id is in camel case.

Node object spelling:
![image](https://cloud.githubusercontent.com/assets/13090472/14430807/a48ab87c-ffc8-11e5-9d5c-fa7b1fee1ac6.png)

HistoryReadValueId spelling:
![image](https://cloud.githubusercontent.com/assets/13090472/14430826/b0c39924-ffc8-11e5-983e-045282ea867f.png)